### DISCUSSION TITLE
Any plans to add the UWG5?

### DISCUSSION THREAD

**Discussion by ursamageor** [robbinjanssen/home-assistant-ojmicroline-thermostat#268] [General]
The UWG5-4999 Has been out for 6 months or so now. I was thinking of getting one, but really only if it's HA compatible.

Has anyone tried to make this integration work with the new UWG5 model?

**Comment by robbinjanssen** [comment]
Hi @3rock618, sorry I have no idea if this will work. As they are closed source there is no documentation about the API they work with.

**Comment by mayoutze** [comment]
I just installed the UWG5, very neat thermostat. I would love it if it could be integrated with HA! Has anything changed since 2024, is the API documented and public now? 
Thank you!
